### PR TITLE
Update process to workflow

### PR DESCRIPTION
**Pull request by antho1404**
Rename process to workflow
related to https://github.com/liteflow-labs/company/issues/54

### CODE REVIEW

File: packages/cli/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 #### Bug fixes
 
+- [#](https://github.com/liteflow-labs/liteflow-js/pull/) Fix confirmation label with correct naming.
```
Comment (antho1404) [inline packages/cli/CHANGELOG.md:11]:
```suggestion
- [#241](https://github.com/liteflow-labs/liteflow-js/pull/241) Fix confirmation label with correct naming.
```

**Review by NicolasMahe — APPROVED** [review]

File: packages/cli/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
 
 #### Bug fixes
 
+- [#241](https://github.com/liteflow-labs/liteflow-js/pull/241) Fix confirmation label with correct naming.
```
Comment (NicolasMahe) [inline packages/cli/CHANGELOG.md:11]:
```suggestion
- [#241](https://github.com/liteflow-labs/liteflow-js/pull/241) Rename process to workflow.
```

**Review by NicolasMahe — COMMENTED** [review]
TODO not bloking:
- [ ] Update README of compiler
- [ ] Rename process to workflow in liteflowrc of compiler

**Comment by antho1404** [comment]
I didn't wanted to do a breaking change on the cli now, it was quite a lot of work. We can work later on a full renaming, compiler and also apis

**Pull request merged**